examples/tutorials/pipeline_end_to_end_tutorial.ipynb

```diff
@@ -0,0 +1,1472 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9a2ba397",
+   "metadata": {},
+   "source": [
+    "# Building Pipelines with Artifacts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b1920421",
+   "metadata": {},
+   "source": [
+    "**Table of Contents**\n",
+    "\n",
+    "- [Intro](#Intro)\n",
+    "- [Demo](#Demo)\n",
+    "    - [Exploration and Pre-processing](#Exploration-and-Pre-processing)\n",
+    "    - [Modeling](#Modeling)\n",
+    "    - [Pipeline Building](#Pipeline-Building)\n",
+    "    - [Running Pipelines Locally](#Running-Pipelines-Locally)\n",
+    "- [Recap](#Recap)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e6dbaf9",
+   "metadata": {},
+   "source": [
+    "## Intro"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "627bce95",
+   "metadata": {},
+   "source": [
+    "### Scenario\n",
+    "\n",
+    "As notebooks become more mature, they may get used like pipelines. For instance, notebooks that do the following things:\n",
+    "\n",
+    "* Process latest data and update dashboards/graphs/charts/reports\n",
+    "* Preprocess new data and dump it to the filesystem for downstream model development\n",
```

Comment:
dump it to the filesystem or db

... something to indicate data isn't restricted to filesystem